packages/terra-form/src/_Utilities.jsx

```diff
@@ -0,0 +1,37 @@
+const autoResizeTextareaHeight = (onChange, baseRowCount) =>
+  (e) => {
+    const textarea = e.currentTarget;
+    const lineHeight = Math.ceil(parseFloat(window.getComputedStyle(textarea).lineHeight, 0));
+
+    /* Smallest Textarea should have at least 2 rows. */
+    const minRows = baseRowCount || 2;
+    textarea.rows = minRows;
+    const rows = Math.ceil((textarea.scrollHeight - textarea.baseScrollHeight) / lineHeight);
+    textarea.rows = minRows + rows;
+
+    if (onChange) {
+      onChange(e);
```

Comment:
Should we pass back the new rows value? Not sure if that could be valuable?

Reply:
I don't think so at the moment. This is designed to be a private utility, and since there isn't an immediate need for it, I don't see the need to return it at the moment.